package.json

```diff
@@ -56,7 +56,7 @@
     "@types/node": "18.11.19",
     "@types/phoenix": "1.5.4",
     "@types/qrcode": "1.5.0",
-    "@types/react": "17.0.53",
+    "@types/react": "^18.0.4",
```

Comment:
Dep should be pinned. And are there no changes in `yarn.lock` when the dep is upgraded?

Reply:
https://github.com/Magickbase/godwoken-explorer-ui/actions/runs/4540819243/jobs/8002232845?pr=928#step:4:23 CI failed because of the inconsistency

Reply:
> Dep should be pinned. And are there no changes in `yarn.lock` when the dep is upgraded?

yep. The version of Dep was fixed and the error was also resolved.
The `yarn.lock` seems to be ignored and the `package-lock` is instead.